client/my-sites/themes/theme-options.js

```diff
@@ -49,9 +50,33 @@ const purchase = config.isEnabled( 'upgrades/checkout' )
 		} ),
 		getUrl: getThemePurchaseUrl,
 		hideForTheme: ( state, themeId, siteId ) => (
+			( config.isEnabled( 'jetpack/pijp' ) && isJetpackSite( state, siteId ) ) ||
+			! getCurrentUser( state ) ||
+			hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) ||
+			! isThemePremium( state, themeId ) ||
+			isThemeActive( state, themeId, siteId ) ||
+			isPremiumThemeAvailable( state, themeId, siteId )
+		)
+	}
+	: {};
+
+const upgradePlan = config.isEnabled( 'upgrades/checkout' )
+	? {
+		label: i18n.translate( 'Upgrade to activate', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Upgrade and activate' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).